Documentation/contributor-guide/review.md

```diff
@@ -0,0 +1,28 @@
+# Review policy
```

Comment:
Rather than a separate file could we add this policy into our existing pr management document https://github.com/etcd-io/etcd/blob/main/Documentation/contributor-guide/triage_prs.md.

Reply:
Sounds good. Will move to there later.

Reply:
Updated & thx